Formula/solr@7.7.rb

```diff
@@ -7,6 +7,11 @@ class SolrAT77 < Formula
   license "Apache-2.0"
   revision 1
 
+  livecheck do
```

Comment:
```suggestion
  # Remove the `livecheck` block (so the check is automatically skipped) once
  # the 7.7.x series is reported as EOL on the first-party downloads page:
  # https://lucene.apache.org/solr/downloads.html#about-versions-and-support
  livecheck do
```

7.7.3 is [reportedly the last version in the 7.7.3 series](https://lucene.apache.org/solr/downloads.html#solr-773) but its status is currently described as ["Previous major version may sometimes receive critical bugfix releases"](https://lucene.apache.org/solr/downloads.html#about-versions-and-support) rather than "End of Life (EOL)".

I think we should add a comment before the `livecheck` block to make it clear that it should be removed when the software becomes EOL and there's no chance of a critical security patch or bug fix in the future. At that point, removing the `livecheck` block will cause it to be automatically skipped again (since it's a versioned formula).

Reply:
yeah, that is a more thoughtful touch. 👍